server/sonar-web/src/main/js/helpers/l10n.js

```diff
@@ -39,6 +39,14 @@ export function translateWithParameters(messageKey: string, ...parameters: Array
   }
 }
 
+export function translatePlural(key: string, parameter: number, pluralKey?: string) {
```

Comment:
this is a good improvement... but only for some languages, for example in Russian there are three plural forms.

I've already planned to work on the subject in [SONAR-7878](https://jira.sonarsource.com/browse/SONAR-7878), let's not mix the subjects here.

For the moment let's simple use `groups(s)`, i.e. `1 group(s)` and `7 group(s)`.